Mathlib/Analysis/Convex/Function.lean

```diff
@@ -1085,6 +1085,51 @@ end OrderedAddCommMonoid
 
 end LinearOrderedField
 
+section OrderIso
+
+variable [OrderedSemiring 𝕜] [OrderedAddCommMonoid α] [SMul 𝕜 α]
+  [OrderedAddCommMonoid β] [SMul 𝕜 β]
+
+theorem strictConvexOn_orderIso_symm (f : α ≃o β) (hf : StrictConcaveOn 𝕜 univ f) :
```

Comment:
Can we put this (and the next three lemmas) in a namespace so that we have dot notation? I think `OrderIso.strictConvexOn` for this, and analogously for the others. Or if you want, `StrictConcaveOn.strictConvexOn`?

Reply:
I went with `OrderIso.strictConvexOn_symm` and so on.